lib/elixir/lib/kernel.ex

```diff
@@ -2198,6 +2198,54 @@ defmodule Kernel do
     :erlang.error(ArgumentError.exception(error_message))
   end
 
+  @doc """
+  Returns true if `term` is a structure; otherwise returns `false`.
```

Comment:
```suggestion
  Returns true if `term` is a struct; otherwise returns `false`.
```
We never use the word "structure".  The only time we use is it in `Kernel.defstructure/2` and it is for something completely different.